tools/image-info

```diff
@@ -64,8 +65,22 @@ def open_image(ctl, image, fmt):
     with tempfile.TemporaryDirectory() as tmp:
         if fmt != "raw":
             target = os.path.join(tmp, "image.raw")
-            subprocess.run(["qemu-img", "convert", "-O", "raw", image, target],
-                           check=True)
+            # A bug exists in qemu that causes the conversion to raw to fail
```

Comment:
Looks reasonable to me. @gicmo, you agree?

Reply:
For stylistic reasons would have probably gone for a:

```
cmd = ["qemu-img", "convert", "-m", "1", "-O", "raw"]
if platform.machine() == 'aarch64':
   cmd += ["-m", "1"]
subprocess.run(cmd + [image, target], ...
```
but I am totally fine with the logic. :+1:

Reply:
Let's leave cleaunps for a possible follow-up then.